service/installer/installer.go

```diff
@@ -56,33 +58,33 @@ func New(config Config) (spec.Installer, error) {
 
 	var err error
 
-	var configurerService configurerspec.Configurer
-	{
+	var configurerServices []configurerspec.Configurer
+	split := strings.Split(config.Viper.GetString(config.Flag.Service.Deployer.Installer.Configurer.Types), ",")
```

Comment:
I don't love `split` as a variable name - `types` would make more sense to me. No blocker.